### ISSUE TITLE
Support actions/cache

### ISSUE THREAD

**Issue by yous** [helaili/jekyll-action#22]
From actions/cache's [examples for Bundler](https://github.com/actions/cache/blob/master/examples.md#ruby---bundler):

``` yml
- uses: actions/cache@v1
  with:
    path: vendor/bundle
    key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
    restore-keys: |
      ${{ runner.os }}-gems-
```
``` yml
- name: Bundle install
  run: |
    bundle config path vendor/bundle
    bundle install --jobs 4 --retry 3
```

**Comment by helaili** [comment]
Thanks @yous, this improvement just got merged through #17

**Comment by yous** [comment]
Nice! Thanks for the notice.
